### DISCUSSION TITLE
conversion script (not gui)

### DISCUSSION THREAD

**Discussion by sobhonium** [sn4k3/UVtools#697] [Scripts]
I have a .sl1s file now. How can I directly convert it to .pwmb in terminal or by python script? I want an equivalent thing like in UVtools GUI 'File+Convert to + ...'. I have a Linux machine.

**Comment by sn4k3** [comment]
See if that helps: https://github.com/sn4k3/UVtools#command-line
After input commands it will give you syntax and help.
Should be easy as: UVtoolsCmd convert input.sl1s pwmb

**Reply by sobhonium** [reply]
Thanks.
